pkg/kv/kvclient/kvcoord/txn_interceptor_pipeliner.go

```diff
@@ -541,6 +564,7 @@ func (tp *txnPipeliner) chainToInFlightWrites(ba *kvpb.BatchRequest) *kvpb.Batch
 						},
 						Txn:            meta,
 						ErrorIfMissing: true,
+						LockStrength:   w.LockStrength,
```

Comment:
Discussed in person — we don't want to send QueryIntentRequests for intents+locks at ignored sequence numbers. Just delete them from `ifWrites` immediately and don't chain on to them.